libs/blocks/ost/ctaTextOption.js

```diff
@@ -23,10 +23,17 @@ const ctaTextOption = {
 
   getSelectedText(searchParameters) {
     const ctaLabel = searchParameters.get('text');
+    if (ctaLabel && this.getCtaFormat(searchParameters) === 'text') {
+      return ctaLabel;
+    }
     return !!ctaLabel && this.ctaTexts.find((label) => label.id === ctaLabel)
       ? ctaLabel
       : this.getDefaultText();
   },
+
+  getCtaFormat(searchParameters) {
+    return searchParameters.get('format') ? searchParameters.get('format') : 'cta';
+  },
```

Comment:
Personal preference, one line arrow functions - you could use that across the whole file
```suggestion
  getCtaFormat: (searchParameters) => (searchParameters.get('format') || 'cta'),
```